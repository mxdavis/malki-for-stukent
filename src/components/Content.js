import React from 'react';
import Fragment from 'render-fragment';

import './ContentParsed.css';
import {greatThings} from '../data/greatThings'
import stukentLogo2 from '../images/stukentLogo2.png';


export const Content = () => (
  <div className="Content">
    <h1> Hi <span role="img" aria-label="Wave">ðŸ‘‹</span> I'm Malki and I would be a great fit for <img src={stukentLogo2} alt="StukentLogo" height='130px'/> </h1>
    <div className="InnerContent">
    {greatThings.map((item, index) => (
        <Fragment key={index}>
          <img
            src={`../images/${item.image}`}
            alt={item.image}
          />
          <h3>{item.text}</h3>
        </Fragment>
    ))}
          </div>
  </div>
)
