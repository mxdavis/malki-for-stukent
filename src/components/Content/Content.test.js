import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import {Content} from './Content';

it('renders without crashing', () => {
  shallow(<Content />);
});

test('Should render Content correctly', () => {
    const tree = renderer.create(
        <Content />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
