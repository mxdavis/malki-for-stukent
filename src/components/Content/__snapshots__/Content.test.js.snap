// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Content correctly 1`] = `
<div
  className="Content"
>
  <h1>
     Hi 
    <span
      aria-label="Wave"
      role="img"
    >
      ðŸ‘‹
    </span>
     I'm Malki and I would be a great fit for 
    <img
      alt="StukentLogo"
      height="130px"
      src="stukentLogo2.png"
    />
     
  </h1>
  <div
    className="InnerContent"
  >
    <img
      alt="code.png"
      src="../images/code.png"
    />
    <h3>
      I love to code, and I'm good at it.
    </h3>
    <img
      alt="ideas.png"
      src="../images/ideas.png"
    />
    <h3>
      I'm a fast learner and I'm excited to learn new things.
    </h3>
    <img
      alt="team.png"
      src="../images/team.png"
    />
    <h3>
      I'm an excellent team player and colleague.
    </h3>
    <img
      alt="personality.png"
      src="../images/personality.png"
    />
    <h3>
      I have a fun personality and great values.
    </h3>
    <img
      alt="success.png"
      src="../images/success.png"
    />
    <h3>
      I work against the odds. I don't give up until I figure it out.
    </h3>
    <img
      alt="reactRuby.png"
      src="../images/reactRuby.png"
    />
    <h3>
      Ruby on Rails and React/Redux are my specialties.
    </h3>
  </div>
</div>
`;
